packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -604,33 +604,43 @@
         "label": {
             "property": "Property",
             "rule": "rule",
-            "repair" :  "recreate message sequences"
+            "string": "string",
+            "number": "number",
+            "boolean": "boolean",
+            "array": "array",
+            "buffer": "buffer",
+            "object": "object",
+            "jsonString": "JSON string",
+            "undefined": "undefined",
+            "null": "null",
+            "repair": "recreate message sequences"
```

Comment:
I think we should add the common types to the main editor.json catalog - `common.type.XYZ` - to maximise their reuse. We already have some of them under `typedInput.type.XYZ`, although those are a special case as the messages are looked-up dynamically.